optuna_dashboard/ts/apiClient.ts

```diff
@@ -58,7 +58,6 @@ const convertTrialResponse = (res: TrialResponse): Trial => {
 interface PreferenceHistoryResponce {
   history: {
     id: string
-    preference_id: string
```

Comment:
How about renaming an existing `id` to `preference_id` to avoid confusion?

Reply:
This id means both history's id and preference's id. Moreover, on the front side, this id will be used more often as a history id. So I think prefer id to preference_id.